BiweeklyMeetings/2022-06-28.md

```diff
@@ -0,0 +1,442 @@
+  * [Review](#review)
+    + [CIP-0044? | Additional factors for NFT market verification](#pr226)
+    + [CIP-0381? | Plutus support for pairings over curve Bls12-381](#pr220)     
+    + [CIP-0050? | Shelleys Voltaire decentralization update](#pr242)
+  * [Discussions](#discussions)
+  * [Issues](#issues)
+  * [Close](#close)
+
+## Summary
+
+Rough transcript of 28/06/22 Editors CIP meeting, to increase transparency and dialogue with the community regarding proposed changes, implementations and considerations.  
+<sub>_Transcript might contain errors or miss pieces - call out issues as needed_
+</sub>  
+Editors meetings are [public](https://discord.gg/36ezQYTvgk), [recorded](https://www.youtube.com/c/CardanoFoundation/videos) and [transcript](https://github.com/cardano-foundation/CIPs/tree/master/BiweeklyMeetings): do join and participate for discussions/PRs of significances to you.  
+
+
+## Editors Meeting Flow
+
+**Triage/Review**: Some CIPs might fall out of grace or not get updated, a CIP that hasn’t seen activity for 3 months should be checked on, and appropriate action taken. Ex: did any of the recent changes obsolete current CIPs? Consider ‘Active’ -> ‘Obsolete’ transitions..  
+
+**Last Check**: Review of the PRIOR meetings Decisions  - if no objection, apply change (effectively a two week lag from decision to action, as a grace period)  
+
+**New CIPs Review**: CIPs up for review should be looked over collectively, with discussion where needed. (on top of the asynchronous reviews)  
+
+PR -> ‘Draft’: Needs format + approval.  
+‘Draft’ -> ‘Proposed’: Needs a PLAN towards Active + implementation.  
+‘Proposed’ -> ‘Active’:  Objective criteria as laid out observed, and consensus agreeing.   
+**Current Discussions**: What the current CIPs discussions are on social media / forums / Discord.  
+
+**Issues**: Review of the open Issues on the CIP GitHub 
+
+**Close**: Recap of actions taken and decisions. List the CIPs that are due for review.  
+
+
+## June 28 2022 Transcript
+**Attending Editors**: Matthias Benkort, Sebastien Guillemot, Frederic Johnson, Robert Phair. 
+**Guests**: Morgan Thomas, Michael Liesenfelt
+
+### Triage
+
+#### PR271
+[PR271 CIP-0027? | Update Proposal: Multiple Royalty Addresses](https://github.com/cardano-foundation/CIPs/pull/271)
+
+**Matthias** -: CIP-0027? | Update Proposal: Multiple Royalty Addresses
+  And that current CIP is already active. It's been used by a few marketplaces. So it doesn't quite make sense to modify it as much. As for the proposal itself, I made already some remark last week. I think the author came back saying yes, true. But that would make for a worse UX. And therefore I would prefer the approach. They have a client in the proposal, which is fair enough, but I still maintain that it should be made as a separate proposal, which is an extension of the existing one rather than modifying what exists and has been used in a while.
```

Comment:
```suggestion
  And that current CIP is already active. It's been used by a few marketplaces. So it doesn't quite make sense to modify it as much. As for the proposal itself, I made already some remark last week. I think the author came back saying: "Yes, true. But that would make for a worse UX. And therefore I would prefer the approach". They have a good rationale in the proposal, which is fair enough, but I still maintain that it should be made as a separate proposal, as an extension of the existing one rather than modifying what exists and has been used in a while.
```